src/client.ts

```diff
@@ -243,21 +243,21 @@ export class Original {
 
   /**
    * listAssets
-   * @param {string} userId userId of the owner of the assets to get
+   * @param {string} userUid uid of the owner of the assets to get
    * @return {Promise<APISearchResponse<Asset[]>>} Returns a list of assets owned by the user, or null data if not found.
    */
-  async getAssetsByUserId(userId: string) {
-    return await this._get<APISearchResponse<Asset[]>>('asset', { user_uid: userId });
+  async getAssetsByUserUid(userUid: string) {
+    return await this._get<APISearchResponse<Asset[]>>('asset', { user_uid: userUid });
   }
 
   /**
    * editAsset
    * @param {string} uid uid of the asset to edit
    * @param {EditAssetParams} asset The details of the asset to be edited
-   * @return {Promise<APIResponse<Uid>>} Returns the uid of the edited asset
+   * @return {Promise<{success: boolean}>} Returns success status of the edit
```

Comment:
This should be `APIResponse<null>` https://docs.getoriginal.com/docs/edit-asset

```
{
    "success": true,
    "data": null
}
```

Reply:
Btw, do we need APISearchResponse ? why not add `| null` to the data on APIResponse?

Reply:
You could also leave APIResponse as is I believe, and just pass `null` as the template arg